prov/rxm/src/rxm_init.c

```diff
@@ -39,12 +39,6 @@
 #include "rxm.h"
 #include "ofi_coll.h"
 
-#define RXM_ATOMIC_UNSUPPORTED_MSG_ORDER (FI_ORDER_RAW | FI_ORDER_ATOMIC_RAW | \
-					  FI_ORDER_RAR | FI_ORDER_ATOMIC_RAR | \
-					  FI_ORDER_WAW | FI_ORDER_ATOMIC_WAW | \
-					  FI_ORDER_WAR | FI_ORDER_ATOMIC_WAR | \
-					  FI_ORDER_SAR | FI_ORDER_SAW)
-
```

Comment:
Shouldn't the FI_ORDER_XXX bits remain since they imply ordering between both RMA and Atomics, and an RxM RMA after an emulated Atomic may not maintain order?

Reply:
I agree with Steve.  FI_ORDER_ATOMIC_XAX ordering appears to be supported.  Atomics are emulated, but RMA uses the msg endpoint RMA capability.  So the FI_ORDER_XAX bits should be here.

Reply:
Right, makes sense. Updated.